Makefile

```
@@ -47,184 +76,98 @@ test-integration-cover:
 		go test -failfast -tags=integration -coverprofile=coverage.out -covermode=count $(addprefix $(PKG)/,$(pkg)) || exit 1;\
 		tail -n +2 coverage.out >> coverage-all.out;)
 
-.PHONY: build_linux
-build_linux:
+.PHONY: build-linux
+build-linux:
 	env GOOS=linux GOARCH=amd64 go build $(APP_FILE)
 
-.PHONY: create-app-ns
-create-app-ns:
-	@echo Creating the namespace ${APP_NAMESPACES}:
-	oc new-project ${APP_NAMESPACES}
+.PHONY: build-master
+build-master:
+	@echo Building operator with the tag: $(IMAGE_MASTER_TAG)
+	operator-sdk build $(IMAGE_MASTER_TAG)
+
+.PHONY: build-release
+build-release:
+	@echo Building operator with the tag: $(IMAGE_RELEASE_TAG)
+	operator-sdk build $(IMAGE_RELEASE_TAG)
+
+.PHONY: push-master
```

```suggestion
.PHONY: push-master-image
```
Push could refer to pushing to a code repo, I think being more explicit that it is an image we're pushing would be clearer

---

👍